static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -46,9 +46,20 @@ import type { Modules } from './types';
 
 const { isDotcomRendering, page } = window.guardian.config;
 
-const assetsPath = page.frontendAssetsFullURL ?? page.assetsPath;
+const decideAssetsPath = () => {
```

Comment:
I don't think we need lines 51 and 52. Also if we test whether `assetsPathPort` is falsy, then if the env variable were to be an empty string for some reason, then it will run the code in the `else` clause.
```suggestion
const decideAssetsPath = () => {
const assetsPathPort = process.env.OVERRIDE_BUNDLE_PORT

if (assetsPathPort) {
		return `http://localhost:${assetsPathPort}/`;
	} else {
		const assetsPath = page.frontendAssetsFullURL ?? page.assetsPath;
		return `${assetsPath}javascripts/commercial/`;
	}
}
```

Reply:
I tried to do a Github code suggestion but I can't work it out yet 😅

Reply:
🤦 Oh yeah, I should've just removed the ternary when we stopped casting to a Number, thanks for spotting! Thanks for the other point as well, makes sense!